### PR TITLE
Add PT-BR automatic translation preface.Rmd

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: preface.Rmd [hunk 1]
```diff
@@ -1,26 +1,26 @@
 ---
-repo-actions: true
+repo-actions: yes
```
Comment (yabellini) [inline preface.Rmd:2]:
```suggestion
repo-actions: true
```

File: preface.pt.Rmd [hunk 2]
```diff
@@ -2,25 +2,25 @@
 repo-actions: true
 ---
 
-# Preface {.unnumbered}
+# Prefácio {.unnumbered}
 
-Welcome! This book is a guide for authors, maintainers, reviewers and editors of rOpenSci.
+Você é bem-vindo! Este livro é um guia para autores, mantenedores, revisores e editores da rOpenSci.
```
Comment (beatrizmilz) [inline preface.pt.Rmd:7]:
```suggestion
Boas vindas! Este livro é um guia para autores, mantenedores, revisores e editores da rOpenSci.
```

**Review by beatrizmilz — COMMENTED** [review]
Exemplo!

**Review by IldySilva — APPROVED** [review]

**Comment by pedropark99** [comment]
The review process is being done at PR #785 in case anyone wants to be the second reviewer.

**Comment by IldySilva** [comment]
@pedropark99 
@yabellini 
Shouldn't this task be removed from the second review list? It's duplicated.

**Comment by yabellini** [comment]
@IldySilva, you are right. I remove it from the project view.  Thanks!

**Pull request merged**